cmd/test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	// "os"
+	// "os/exec"
+
+	// log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+//var HELM_DEFAULT_GITHUB = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.helm"
+
+// createCmd represents the create command
+var testCommand = &cobra.Command{
```

Comment:
what's the purpose of this?

Reply:
test command was used to unit test functions with "condo_cli test". I will clean it up but I still think it's worth keeping in the repo for now since we still have a lot of development to do.